stl/src/ppltasks.cpp

```diff
@@ -29,7 +29,7 @@ static GUID const Local_IID_ICallbackWithNoReentrancyToApplicationSTA = {
 #if defined(_CRT_APP) && defined(_DEBUG)
 extern "C" NTSYSAPI _Success_(return != 0) WORD NTAPI
     RtlCaptureStackBackTrace(_In_ DWORD FramesToSkip, _In_ DWORD FramesToCapture,
-        _Out_writes_to_(FramesToCapture, return) PVOID* BackTrace, _Out_opt_ PDWORD BackTraceHash);
+        _Out_writes_to_(FramesToCapture, return) PVOID* BackTrace, _Out_opt_ PDWORD BackTraceHash) noexcept;

```

Comment:
This function is definitely not ours - should we be changing our handcrafted declaration?

Reply:
Ack, no.